### DISCUSSION TITLE
Should vti device name must start with the prefix "vti" for route based ipsec?

### DISCUSSION THREAD

**Discussion by redbrick9** [strongswan/strongswan#1919] [Q&A] [answered]
Dear maintainers,

For some reasons, I have to create vti device with the name which begin with non-vti prefix, so I would like know if the vti device with this kind of name works as expected. Thanks.

Best,
- Brick

**Comment by tobiasbrunner** [comment, marked as answer]
The name doesn't really matter, see [the docs](https://docs.strongswan.org/docs/5.9/features/routeBasedVpn.html#_vti_device_management) for details.
